src/basic_commands/block_size.md

```diff
@@ -28,13 +28,12 @@ The `b` command is used to change the block size:
 The `bf` command is used to change the block size to value specified by a flag. For example, in symbols, the block size of the flag represents the size of the function.
 ```
 [0x00000000]> bf sym.main    ; block size = sizeof(sym.main)
-[0x00000000]> pd @ sym.main  ; disassemble sym.main
+[0x00000000]> pD @ sym.main  ; disassemble sym.main
 ...
 ```
 
-You can combine two operations in a single one (`pdf`):
+You can combine two operations in a single `pdf` command, plus without messing around global block size:
```

Comment:
Not clear to me what thsi paragraph means i think this can be rephrased better

Reply:
Now I know just enough to explain what is actually going on in `pdf`. Rewording is coming.